style.css

```diff
@@ -0,0 +1,65 @@
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 375px;
+  margin: 0 37%;
+}
```

Comment:
- [ ] Consider avoiding the use of width in these cases because its are affecting the responsiveness of the website